manatest/testfile.go

```diff
@@ -215,9 +226,21 @@ func (testFile *TestFile) MakeTestHeaders(projFile *ProjectFile) []TestHeader {
 	return headers
 }
 
-// MaketestURL Prepares HTTP URL for the test but replacing necessary variables.
+// MakeTestURL Prepares HTTP URL for the test but replacing necessary variables.
 func (testFile *TestFile) MakeTestURL(projFile *ProjectFile) string {
-	return ReplaceVarsInTestURL(testFile.URL, &projFile.Globals)
+	v := url.Values{}
+	for key, value := range testFile.Params {
+		key = ReplaceVarsInHeader(key, &projFile.Globals)
+		value = ReplaceVarsInHeader(value, &projFile.Globals)
+		v.Set(key, value)
```

Comment:
We should probably look into properly encoding these? Or  make sure we are passing them into Resty down the line and that Resty is taking care of it.

Reply:
I think net/url `Values` should take care of this: https://play.golang.org/p/kP-akul-4-Y

Reply:
👍 Thank you for confirming